app/Http/Controllers/WikiController.php

```diff
@@ -79,6 +80,14 @@ public function create(Request $request): \Illuminate\Http\Response
                 'value' => Str::random(64),
             ]);
 
+            // Enable elasticsearch for new wikis by default
+            // T285541
+            WikiSetting::create([
```

Comment:
For now I don't think we want to do this!
We should leave the default to be false.
We do however still want to be able to call the API that we made, when the thing is disabled.